ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Monitor.hs

```diff
@@ -206,6 +214,11 @@ connections PeerSelectionActions{
             bigLedgerPeersDemotions = nonLocalDemotions
                    `Map.restrictKeys` bigLedgerPeers
 
+            -- Peers in this state won't be able to be promoted nor demoted.
+            inProgressDemoteToCold' =
+                 inProgressDemoteToCold <> Map.keysSet demotedToCooling
+                  Set.\\ Map.keysSet demotedToCold
```

Comment:
`Set.\\` has higher fixity than `<>`, so we need:
```suggestion
                 (inProgressDemoteToCold <> Map.keysSet demotedToCooling)
                 Set.\\ Map.keysSet demotedToCold
```

Reply:
or the other way around, e.g. `inProgressDemoteToCold Set.\\ Map.keysSet demotedToCold <> Map.keysSet demotedToCooling`.